docs/components/textarea-field/index.md

```diff
@@ -0,0 +1,111 @@
+# Textarea Field
+
+Provides an underlying `<textarea>` element building on top of the Field component.
+
+## Label
+
+Provide a string to `@label` to render the text into the `<label>` of the Field.
+
+## Hint
+
+Provide a string to `@hint` to render the text into the Hint section of the Field. This is optional.
+
+## Error
+
+Provide a string to `@error` to render the text into the Error section of the Field. This is optional.
+
+## Value and onChange
+
+To tie into the input event, provide `@onChange`. `@onChange` will return two arguments, the first being the value, while the second being the raw event object. It's most common to use this in combination with `@value` which will set the value for the textarea based on the input received from the change event.
```

Comment:
Nit, suggestion: I would bullet `value` and the `raw event object`, something like:

```
 `@onChange` will return two arguments:
 - the value of the input
 - the raw event object
```

This is because it gets buried in the middle of the paragraph and some folks are unlikely to read this.

Reply:
Thanks for the suggestion! Applied!